### PR TITLE
feat: add token dialog ui

### PR DESCRIPTION
**Pull request by jhasuraj01**

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-5006 Add Token on CySync/CLI](https://app.clickup.com/t/9002019994/PRF-5006)

File: packages/cysync-core/src/dialogs/AddToken/context/index.tsx [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+  selectAccounts,
+  selectLanguage,
+  useAppDispatch,
+  useAppSelector,
+} from '~/store';
+
+import { CoinIcon } from '~/components';
+import { AddTokenCongrats, AddTokenSelectionDialog } from '../Dialogs';
+
+export type AddTokenStatus = 'idle' | 'device' | 'sync' | 'done';
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/AddToken/context/index.tsx]: Unused variable

File: packages/cysync-core/src/pages/MainApp/Wallet/index.tsx [hunk 2]
```diff
@@ -60,7 +61,9 @@ export const Wallet: FC = () => {
           text={lang.strings.wallet.accountMissing.text}
           subText={lang.strings.wallet.accountMissing.subText}
           $buttonOne={lang.strings.buttons.addAccount}
+          $buttonTwo={lang.strings.buttons.addToken}
```
Comment (irshadCypherock) [inline packages/cysync-core/src/pages/MainApp/Wallet/index.tsx]: I don't think the user can add token if he does not have an account

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: packages/cysync-core/src/dialogs/AddToken/context/index.tsx (hunk 1 above)
Comment (jhasuraj01) [reply packages/cysync-core/src/dialogs/AddToken/context/index.tsx]: removed `AddTokenStatus`

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**